sdk/storage/inc/blobs/internal/protocol/blob_rest_client.hpp

```diff
@@ -83,7 +137,8 @@ class BlobRestClient
       {
         // TODO: ContainerUrl initialization.
         info.Etag = Common::GetHeaderValue(response.getHeaders(), Common::k_HEADER_ETAG);
-        info.LastModified = Common::GetHeaderValue(response.getHeaders(), Common::k_HEADER_LAST_MODIFIED);
+        info.LastModified
+            = Common::GetHeaderValue(response.getHeaders(), Common::k_HEADER_LAST_MODIFIED);
```

Comment:
>GetHeaderValue [](start = 22, length = 14)

Question: Is your maximum length set to 80? Do you think this is the value we should use in CLANG formatter?

Reply:
I  think it's 100